sentry-sample/src/main/AndroidManifest.xml

```diff
@@ -58,6 +58,9 @@
 <!--    how to enable session tracking-->
     <meta-data android:name="io.sentry.session-tracking.enable" android:value="true" />
 
+<!--    how to enable auto session tracking integration-->
```

Comment:
I'm not sure I follow what would be the use case for `io.sentry.session-tracking.enable`

Reply:
In other words: Doesn't the new field replace the old one?